addusers.py

```diff
@@ -201,7 +203,106 @@ def exists_in_keystone(check_thing, keystone_things):
         return None
 
 
-def parse_rows(rows, select_user=None):
+def remove_duplicates(rows, UC, PC, EC):
```

Comment:
Before we call the delete_rows() function by spreadsheet function, I checked if there are any duplicates in rows to avoid bug #23.